lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -3567,7 +3570,7 @@ void _testRoute() {
     tester.apply();
 
     expect(capturedActions, isEmpty);
-    expect(domDocument.activeElement, domDocument.body);
+    expect(domDocument.activeElement, flutterViewRoot);
```

Comment:
Please @yjbanov TAL at this new change. I think it makes sense, because we're actually transferring the focus to the flutter view and not the body, but I don't know what this can break :)